server/src/main/java/org/opensearch/action/search/SearchContextId.java

```diff
@@ -111,7 +111,7 @@ public static SearchContextId decode(NamedWriteableRegistry namedWriteableRegist
             }
             return new SearchContextId(Collections.unmodifiableMap(shards), Collections.unmodifiableMap(aliasFilters));
         } catch (IOException e) {
-            throw new IllegalArgumentException(e);
+            throw new IllegalArgumentException("Cannot parse pit id", e);
```

Comment:
This is used only for PIT - scroll uses TransportSearchHelper encode and decode. But will remove PIT as part of exception and just keep id so that its generic